packages/node/vro-language-server/src/server/core/HintLookup.ts

```diff
@@ -139,9 +139,7 @@ export class HintLookup implements Disposable {
         for (const api of this.scriptingApi.global) {
             for (const cls of api.classes) {
                 const hasConstructors = !!cls.constructors && cls.constructors.length > 0
-                if (filter.isInstantiable === undefined) {
-                    result.push(cls)
-                } else if (hasConstructors === filter.isInstantiable) {
+                if (filter.isInstantiable === undefined || hasConstructors === filter.isInstantiable) {
```

Comment:
Let's add extra brackets just for safety of conditions
if ((filter.isInstantiable === undefined) || (hasConstructors === filter.isInstantiable))